src/main/java/com/mineboundteam/minebound/magic/UtilitySpells/ShieldUtilitySpell.java

```diff
@@ -0,0 +1,150 @@
+package com.mineboundteam.minebound.magic.UtilitySpells;
+
+import com.mineboundteam.minebound.MineBound;
+import com.mineboundteam.minebound.config.IConfig;
+import com.mineboundteam.minebound.item.armor.ArmorTier;
+import com.mineboundteam.minebound.magic.PassiveSpellItem;
+import net.minecraft.ChatFormatting;
+import net.minecraft.nbt.CompoundTag;
+import net.minecraft.network.chat.Component;
+import net.minecraft.network.chat.TextComponent;
+import net.minecraft.world.damagesource.EntityDamageSource;
+import net.minecraft.world.entity.Entity;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.TooltipFlag;
+import net.minecraft.world.level.Level;
+import net.minecraftforge.common.ForgeConfigSpec;
+import net.minecraftforge.event.TickEvent;
+import net.minecraftforge.event.entity.living.LivingAttackEvent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.event.config.ModConfigEvent;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.List;
+
+@Mod.EventBusSubscriber(modid = MineBound.MOD_ID)
+public class ShieldUtilitySpell extends PassiveSpellItem {
+
+    private final int manaCost;
+    private final int totalHits;
+    private final int recovCooldown;
+
+    public ShieldUtilitySpell(Properties properties, ShieldUtilitySpellConfig config) {
+        super(properties, config.LEVEL);
+
+        this.manaCost = config.MANA_COST.get();
+        this.totalHits = config.TOTAL_HITS.get();
+        this.recovCooldown = config.RECOV_TICKS.get();
+    }
+
+    @SubscribeEvent
+    public static void triggerSpell(LivingAttackEvent event) {
```

Comment:
Do we have cards for playing sounds on spell cast?

Reply:
I don't think we do. Not sure we even considered custom sounds and sound triggers in the SRS.

Reply:
I think we should at least for the shield spell. It works but difficult to verify it works for 20 attack etc

Reply:
I thought doing something like this would be unnecessarily complicated, turns out it's not. Pushed.